entrypoint.sh

```diff
@@ -7,6 +7,12 @@ slug_ref() {
          | cut -c1-63
 }
 
+short_sha(){
+  echo "$1" \
+        | cut -c1-8
+}
+
 echo ::set-env name=GITHUB_REF_SLUG::"$(slug_ref "$GITHUB_REF")"
 echo ::set-env name=GITHUB_HEAD_REF_SLUG::"$(slug_ref "$GITHUB_HEAD_REF")"
 echo ::set-env name=GITHUB_BASE_REF_SLUG::"$(slug_ref "$GITHUB_BASE_REF")"
+echo ::set-env name=GITHUB_SHORT_SHA::"$(short_sha "$GITHUB_SHA")"
```

Comment:
I prefer `GITHUB_SHA_SHORT` to be compliant with `slug` one

```suggestion
echo ::set-env name=GITHUB_SHA_SHORT::"$(short_sha "$GITHUB_SHA")"
```